jujugui/static/gui/src/app/init.js

```diff
@@ -261,6 +262,16 @@ class GUIApp {
       hideDragOverNotification: this._hideDragOverNotification.bind(this)
     });
 
+    this.megaWatcherModels = null;
+    this.megaWatcher = new MegaWatcher({
+      changeEvent: '_rpc_response',
+      onChange: models => {
+        this.megaWatcherModels = models;
+        this.state.dispatch();
```

Comment:
When the deltas change then store the new models and dispatch to update the components.